EAD/EAD-Session2/validation.js

```diff
@@ -0,0 +1,76 @@
+function nameValidation(){
+    var first_name = document.getElementById("first_name");
+    var last_name = document.getElementById("last_name");
+    var latters = /^[A-Za-z]+$/;
+    if(first_name.value.length <= 2 || last_name.value.length <= 2){
+        debugger;
+        alert("Please insert a valid name");
+        return false;
+    }
+    if(first_name.value.match(latters || last_name.value.match(latters))){
+        return true;
+    }
+    else{
+        alert("Please insert a valid name");
```

Comment:
if you are showing same message then why two if ? better is to throw exact message that what is going wrong.